CHANGELOG.md

```diff
@@ -1,579 +1,435 @@
-   * Implement onTouched logic.
-   * Fix disabled state to match Material Spec which is a light grey not a
-     certain opacity of the checkbox.
- * Material Dialog:
-   * Remove `preserveWhitespace: false`.
- * Material Expansionpanel:
-   * Smooth appearance/disappearance of expansion panel contents.
-   * Add mixin to allow change the panel name min width.
-   * Remove `preserveWhitespace: false`.
- * Material Input: Fix unexpected `)` in selector for
-   `MaterialInputDefaultValueAccessor`.
- * Material Popup:
-   * Listen to DomService.trackLayoutChange streams for overlayRepositionLoop
-     popups.
- * Material Progress: Fall back to the non-optimized animation if the width is 0
-   after view init.
- * Material Select:
-   * Add `showButtonBorder` input to Material Dropdown Select component.
-   * Cleanup unused styles.
- * Material Tabs: Increase space between labels.
- * Material Tooltip: Fix material-paper-tooltip content that are change detected
-   and using deferred content by having the visible property be set in an async
-   manner so that the value isn't added in the middle of a change detection
-   loop.
- * Material Tree: Clear filter inside the popup when MaterialTreeDropdown
-   closes.
- * Material Yes/No Buttons: Update material-yes-no-buttons mixins to use CSS
-   shimming.
- * Scorecard: Add a tooltip field.
- * Use generic type for table selection model `SelectableGetter`.
- * Add color contrast ratio utility.
- * Remove deprecated box-sizing styles.
- * Deprecate `ManagedZone` and replace with `NgZone`.
- * Remove unnecessary uses of `::ng-deep`.
- * Add `visibility: Visibility.none` to most components for reduced code size.
- * Update documentation.
+> 5.0.0-beta. The alpha tag represents the evolving nature of the AngularDart
+> API, not code quality (5.0.0-beta is used in production Google apps).
+
+Please note that an up-to-date copy of `dev` channel Dart SDK is required (at
+least `2.0.0-dev.65` as of this writing) to use this version with AngularDart
+v5.0.0-beta.
+
+Pub _transformers_ are no longer used. Instead, use the new
+[webdev](https://pub.dartlang.org/packages/webdev) CLI, or, for advanced
+users, the [build_runner](https://pub.dartlang.org/packages/build_runner) CLI.
+
+Please see the Dart 2 [migration guide](https://webdev.dartlang.org/dart-2) for
+more details.
+
+### Updated Dependencies
+* angular: 5.0.0-beta
+* angular_forms: 2.0.0-beta
+
+### New Components
+* Material Stacking Drawer
+* Material Stepper
+* Material Slider
+* Material Date/Time Picker components
+* Material Menu components
+
+### Component Updates
+
+#### App Layout
+* Update Material Header `z-index` to 1.
+* Add support for 0 or 2 drawers.
+* Add mixin for permanent/persistent drawers and fix mixin for temporary
+  drawers.
+* Add mixin for drawer width.
+* Provide list-item changes as a mixin to be used in components that wrap drawer
+  contents.
+
+#### Button Decorator
+* Allow role to be customized.
+
+#### Glyph
```

Comment:
Is \<glyph> not deprecated? If so, maybe mention it, or consider not mentioning changes?

Reply:
Done.